Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -516,6 +516,12 @@ protected virtual void Awake()
 
         private void OnEnable()
         {
+            if(!Enabled)
+            {
+                // Re-enable Interactable
+                SetDisabled(false);
```

Comment:
Ah, so I think this means that if interactable's Enabled property is false, then when you enable the entire component, this will also set interactable's Enabled = true. Can you call this out in the comments for the Enabled property, so people know what behavior to expect?